addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3188,7 +3188,24 @@ export class OdooEditor extends EventTarget {
         this.deleteRange();
     }
     _onClipboardCopy(clipboardEvent) {
-        if (this.isSelectionInEditable()) {
+        const hasPartiallySelectedTable = hasTableSelection(this.editable) &&
+            ![...this.editable.querySelectorAll('.o_selected_table td')].every(td => (
+                !closestElement(td, 'table').classList.contains('o_selected_table') || // ignore nested
+                td.classList.contains('o_selected_td')
+            ));
+        if (hasPartiallySelectedTable) {
+            clipboardEvent.preventDefault();
+            const table = this.editable.querySelector('.o_selected_table');
```

Comment:
And you have to re-querySelector anyway so I suggest to store whatever information that can be reused in separate variables. That will make the computation of the `hasPartiallySelectedTable` condition easier to read and you'll avoid querySelecting the same nodes multiple times later on.